src/reference/v8.json

```diff
@@ -3286,6 +3286,24 @@
           }
         }
       },
+      "global-state": {
+        "doc": "Retrieves a property value from the global state. Returns null if the requested property is missing.\n\n - [Setting global state](https://maplibre.org/maplibre-gl-js/docs/examples/global-state/)",
+        "group": "Lookup",
+        "example": {
+          "syntax": {
+            "method": ["string"],
+            "result": "value"
+          },
+          "value": ["global-state", "someProperty"]
+        },
+        "sdk-support": {
+          "basic functionality": {
+            "js": "TBD",
```

Comment:
Please add relevant issues link here, see other places related to web-native parity.